tests/js/common/aql/aql-arangosearch-inverted-index-search-alias-community.js

```diff
@@ -713,11 +714,14 @@ function IResearchInvertedIndexSearchAliasAqlTestSuiteCommunity() {
             ];
 
             db._create("failCollection");
-            for (let i = 0; i < failDefinitions.length; i++) {
-                let d = failDefinitions[i];
+            for (let i = 1; i <= failDefinitions.length; i++) {
+                if (i === 1 && isEnterprise) {
+                    continue;
+                }
+                let d = failDefinitions[i - 1];
                 try {
                     db.failCollection.ensureIndex(d);
-                    assertTrue(false);
+                    assertFalse(i);
```

Comment:
NIT that is more clear
```suggestion
                    fail();
```